pages/blog/beyond-boundaries.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Beyond Boundaries - Empowering Through OSS Mentorship'
+date: 2023-07-12T06:00:00+01:00
+type: Community
+tags:
+  - Mentorships
+  - OSS
+  - AsyncAPI
+cover: /img/posts/beyond.webp
+weight: 10
+authors:
+  - name: Azeez Elegbede
+    photo: /img/avatars/ace.webp
+    link: https://twitter.com/_acebuild
+    byline: AsyncAPI Preacher
+---
+
+In 2022 we piloted our own very innovative mentorship program that focuses on open-source software(OSS). We envisioned a platform where aspiring developers could learn, grow, and contribute to exciting projects while receiving guidance from experienced mentors.
+
+## Our Backstory
+
+At AsyncAPI, we value giving back to the community, as it gives us a sense of purpose as a community-first organization, and we are always looking for ways to participate in Opensource mentorship programs like Google Summer of Code(GSoC), Google Season of Docs(GSoD) and more because the fulfilling feeling of giving back and contributing to society gotten from this programs is unparalleled.
+
+As an open-source organization, we participate so that we can collaborate with a broader reach of open-source enthusiasts and help bring more folks into the open-source world(which we are already doing). Furthermore, our reason for participating in these programs comes from our passion for open source and the care to support and encourage open-source contributors by building and participating in mentorships.
```

Comment:
```suggestion
As an open-source organization, we participate to collaborate with a broader reach of open-source enthusiasts and help bring more folks into the open-source world(which we are already doing). Furthermore, our reason for participating in these programs comes from our passion for open source and the care to support and encourage open-source contributors by building and participating in mentorships.
```

Reply:
The word "open-source" is repeated many times. This is my proposal:

"As an open-source organization, we participate to collaborate with a broader reach of enthusiasts and help bring more folks into this world(which we are already doing). Furthermore, our reason for participating in these programs comes from our passion for open source and the care to support and encourage contributors by building and participating in mentorships."